src/utils/StringUtils.h

```diff
@@ -123,7 +137,7 @@ inline std::string RightTrim(std::string str, char c)
 inline std::string RightTrim(std::string str)
```

Comment:
Sorry for delayed response.
I wanted to double check that those changes won't break the other parts of the library where we need to deal with unicode strings but then pulled into different directions. 

Are we sure that this "unsigned char" based IsSpace() function can operate reliably with UTF-8 encoded, potentially multi-byte characters? Especially the RightTrim() feels dangerous which is used in places like GetAbsoluteSearchPaths() etc.

Reply:
Hmm with a closer look, that doesn't seem to be using this overload. So the current use-case wise we're safe I guess.

Reply:
Yes this would work correctly with UTF-8 too, as in it would detect ASCII spaces just fine. UTF-8 is a full superset of 7-bit ASCII; any "longer than one byte" character bytes in there all have the highest bit set.

Reply:
Yup, you are right. MSB can be 0 only for the first byte of the multi-byte character. 
All clear!